hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -955,8 +955,14 @@ final class VCFLine(val line: String, arrayElementsRequired: Boolean,
             rvb.addBoolean(true)
         } else
           rvb.addBoolean(true)
-      } else
-        parseAddInfoField(rvb, c.infoFieldTypes(idx))
+      } else {
+        try {
+          parseAddInfoField(rvb, c.infoFieldTypes(idx))
+        } catch {
+          case e: VCFParseError => parseError(s"error while parsing info field '$key': ${ e.msg }")
+          case e => throw e
```

Comment:
Delete this line. Don't catch something just to throw it.

Reply:
This is also the cause of your compilation error. Scala views doing this as a mistake.